snowflake/datadog_checks/snowflake/__init__.py

```diff
@@ -1,6 +1,14 @@
 # (C) Datadog, Inc. 2020-present
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+
+# workaround for problem of platform.libc_ver() doesn't consider
+# in case of sys.executable returns empty string on Python 3.8 or later
+# Python issue link: https://bugs.python.org/issue42257
+import sys
+if not sys.executable:
```

Comment:
```suggestion

if not sys.executable:
```

CI is failing due to style, this will fix

Reply:
Thank you!